### PR TITLE
2019-12-03

### PR DESCRIPTION
**Pull request by valeIT**

### CODE REVIEW

File: _posts/2019-12-04-bitrise-ios.md
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: caan - > can

Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: releaase - > release

Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: Something wrong with yoor A key?

File: _posts/2019-12-04-bitrise-ios.md
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: pusing - > pushing

Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: requedst - > request

Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: wil - > will

File: _posts/2019-12-04-bitrise-ios.md
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: Missing "." at the end

Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: Add them everywhere...

File: _posts/2019-12-04-bitrise-ios.md
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: What's the slash for

File: _posts/2019-12-04-bitrise-ios.md
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: passes - > pass?

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: bette - > better

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md:24]: Babyyyyy

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: issue - > issues

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: it - > them

Comment (MsSlacker) [reply _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: wants - > want

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md:50]: Nothing about story or the characters?

File: _posts/2019-12-06-website-categories.md
Comment (MsSlacker) [inline _posts/2019-12-06-website-categories.md]: E?

Comment (MsSlacker) [reply _posts/2019-12-06-website-categories.md]: CAPITALISE I

File: _posts/2019-12-06-website-categories.md
Comment (MsSlacker) [inline _posts/2019-12-06-website-categories.md]: whole point of that a category - > missing what

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-12-04-bitrise-ios.md
Comment (MsSlacker) [reply _posts/2019-12-04-bitrise-ios.md]: Continuous Integration.......

File: _posts/2019-12-04-bitrise-ios.md
Comment (valeIT) [reply _posts/2019-12-04-bitrise-ios.md]: Got automatically inserted by the editor, no idea why

File: _posts/2019-12-04-bitrise-ios.md
Comment (valeIT) [reply _posts/2019-12-04-bitrise-ios.md]: > Something wrong with yoor A key?

Something is wrong with every key. It is called buttefly keyboard... =(

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (valeIT) [reply _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md:24]: What a worthy comment... :P

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (valeIT) [reply _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md:50]: Can't really say anything about the story until I finish it. Same for the characters I guess. ..?

File: _posts/2019-12-06-website-categories.md
Comment (valeIT) [reply _posts/2019-12-06-website-categories.md]: > E?

what?

Comment (MsSlacker) [reply _posts/2019-12-06-website-categories.md]: "I do not want my blog to be e thematic." what's e thematic?

File: _posts/2019-12-06-website-categories.md
Comment (MsSlacker) [inline _posts/2019-12-06-website-categories.md]: what i find - > this I also needs capitalization!

**Review by MsSlacker — CHANGES_REQUESTED** [review]

File: _posts/2019-12-06-website-categories.md
Comment (valeIT) [reply _posts/2019-12-06-website-categories.md]: oh ok

**Comment by valeIT** [comment]
Wait until #104  is merged before merging this.

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (MsSlacker) [reply _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md:50]: Well you finished it!

**Review by MsSlacker — APPROVED** [review]

**Review by MsSlacker — APPROVED** [review]

**Review by MsSlacker — DISMISSED** [review]
Starting most of them with "here" is not really good tho.

File: _posts/2019-12-04-bitrise-ios.md
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: uploading - > upload

File: _posts/2019-12-04-bitrise-ios.md
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: se - > set

File: _posts/2019-12-04-bitrise-ios.md
Comment (MsSlacker) [inline _posts/2019-12-04-bitrise-ios.md]: fo0rced? Lol

**Review by MsSlacker — CHANGES_REQUESTED** [review]

**Comment by valeIT** [comment]
is it ok?

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: experience - > experiencing

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: subcription - > subscription

File: _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md
Comment (MsSlacker) [inline _posts/2019-12-05-star-wars-jedi-fallen-order-first-impression.md]: will - > with

**Comment by valeIT** [comment]
@MsSlacker

**Review by MsSlacker — APPROVED** [review]

**Pull request merged**
